front-end/src/components/Navbar/Navbar.js

```diff
@@ -0,0 +1,165 @@
+import React, { useState } from 'react'
```

Comment:
You please add the Nav bar to the proposal create page (the search element should not be displayed here)